packages/wdio-local-runner/src/replQueue.ts

```diff
@@ -19,16 +30,16 @@ export default class ReplQueue {
             return
         }
 
-        const { childProcess, options, onStart, onEnd } = this.repls.shift()
-        this.runningRepl = new WDIORepl(childProcess, options)
+        const { childProcess, options, onStart, onEnd } = this.repls.shift() || ({} as Repl)
```

Comment:
The or, should not be needed I think?
```suggestion
        const { childProcess, options, onStart, onEnd } = this.repls.shift()
```